docs/src/main/paradox/scala/http/server-side/server-https-support.md

```diff
@@ -1,20 +1,24 @@
 # Server-Side HTTPS Support
```

Comment:
I think it is because of the path or my preference for taking the Java wordings.